app/src/main/java/com/sunlightlabs/android/congress/BillSearch.java

```diff
@@ -62,10 +62,13 @@ public void setupPager() {
 					this, SuggestionsProvider.AUTHORITY, SuggestionsProvider.MODE);
 	        suggestions.saveRecentQuery(query, null);
 			
-			ActionBarUtils.setTitle(this, "Bills matching \"" + query + "\"", new Intent(this, MenuBills.class));
-			ActionBarUtils.setTitleSize(this, 16);
+			ActionBarUtils.setTitle(this, "Recent bills matching \"" + query + "\"", new Intent(this, MenuBills.class));
+			ActionBarUtils.setTitleSize(this, 14);
 			adapter.add("bills_recent", R.string.search_bills_recent, BillListFragment.forSearch(query, BillListFragment.BILLS_SEARCH_NEWEST));
-			adapter.add("bills_relevant", R.string.search_bills_relevant, BillListFragment.forSearch(query, BillListFragment.BILLS_SEARCH_RELEVANT));
+//			adapter.add("bills_relevant", R.string.search_bills_relevant, BillListFragment.forSearch(query, BillListFragment.BILLS_SEARCH_RELEVANT));
+
+			// if restoring relevance tab, remove below line
+			findViewById(R.id.pager_titles).setVisibility(View.GONE);
```

Comment:
The PP API doesn't have the concept of ordering search results by relevance (see https://github.com/propublica/congress-api-docs/issues/73 for my feature request), and so this tab is removed from search results.